src/Grpc.Net.Client/GrpcChannel.cs

```diff
@@ -360,6 +485,52 @@ public static GrpcChannel ForAddress(Uri address, GrpcChannelOptions channelOpti
             return new GrpcChannel(address, channelOptions);
         }
 
+#if SUPPORT_LOAD_BALANCING
+        /// <summary>
+        /// Allows explicitly requesting channel to connect without starting an RPC.
+        /// Returned task completes once <see cref="State"/> Ready was seen.
+        /// There is no need to call this explicitly unless your use case requires that.
+        /// Starting an RPC on a new channel will request connection implicitly.
+        /// </summary>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <returns></returns>
+        public Task ConnectAsync(CancellationToken cancellationToken = default)
+        {
+            return ConnectionManager.ConnectAsync(waitForReady: true, cancellationToken);
+        }
+
+        ISubchannelTransport ISubchannelTransportFactory.Create(Subchannel subchannel)
+        {
+#if NET5_0_OR_GREATER
+            var isTcpTransport = HttpHandlerType == HttpHandlerType.Default;
+
+            if (isTcpTransport && SocketsHttpHandler.IsSupported)
+            {
+                return new ActiveSubchannelTransport(subchannel, TimeSpan.FromSeconds(5));
+            }
+#endif
+
+            return new PassiveSubchannelTransport(subchannel);
```

Comment:
because currently SUPPORT_LOAD_BALANCING is only set on net50, it looks like this branch is never used, and so the PassiveSubchannelTransport is also never used. 
If that's the case, let's exclude it from this PR to avoid bloating it with dead code. It can always be added in a follow up PR once it's actually useful.